README.md

```diff
@@ -5,3 +5,12 @@ qgepplugin
 This plugin is part of the QGEP Wastewater project for QGIS.
 
 [![Build Status](https://travis-ci.org/QGEP/qgepplugin.svg?branch=master)](https://travis-ci.org/QGEP/qgepplugin)
+
+dev
+---
+
+Code style is enforced with pre-commit. To use :
+```
+pip install pre-commit
+pre-commit install
```

Comment:
Does this need to be run from a particular folder?

Reply:
The first one no (you can install pre-commit globally or in the current venv), the second one in the git repo (installs pre-commit hook)